apps/docs/components/scale/Scale.tsx

```diff
@@ -20,47 +24,56 @@ export const Scale = ({
   valueLabel?: string;
   values: Record<string, string>;
 }) => (
-  <Box maxH="sm" overflow="auto">
-    <Table>
-      <thead>
-        <tr>
-          <Th w="80">{keyLabel}</Th>
-          <Th w="128">{valueLabel}</Th>
-          {!hidePixels && <Th w="128">Pixels</Th>}
-          {!hidePreview && (
-            <Box asChild display={["none", "table-cell"]}>
-              <Th />
-            </Box>
-          )}
-        </tr>
-      </thead>
-      <tbody>
-        {Object.entries(values)
-          .sort(([a], [b]) => {
-            const aMatch = a.match(/^([0-9.]+)$/);
-            const aNum = aMatch === null ? NaN : parseFloat(aMatch[1]);
-            const bMatch = b.match(/^([0-9.]+)$/);
-            const bNum = bMatch === null ? NaN : parseFloat(bMatch[1]);
-            if (isNaN(aNum) && isNaN(bNum)) return 0;
-            if (isNaN(aNum)) return 1;
-            if (isNaN(bNum)) return -1;
-            return aNum - bNum;
-          })
-          .map(([name, size]) => (
-            <Tr key={name}>
-              <Td>{name}</Td>
-              <Td>{size}</Td>
-              {!hidePixels && <Td>{px(size)}</Td>}
-              {!hidePreview && (
-                <Box asChild display={["none", "table-cell"]}>
-                  <Td>
-                    <Box bg="purple.500" h="16" style={{ width: size }} />
-                  </Td>
-                </Box>
-              )}
-            </Tr>
-          ))}
-      </tbody>
-    </Table>
-  </Box>
+  <Table maxH="sm" overflow="auto">
+    <thead>
+      <tr>
+        <Th position="sticky" style={headerBg} top="0" w="80">
```

Comment:
allow scale table to have sticky headers:

https://github.com/optimizely-axiom/optiaxiom/assets/758244/2dae5672-7958-4da5-a3e6-5ba5c8892ae4